docs/help/contributor/style-guide/documenting-code.md

```diff
@@ -0,0 +1,212 @@
+# Documenting Code
+
+??? tip "Looking for a template?"
+    //todo
+
+## Documenting Commands
+
+>**The standard line is “X the Y by running:”**
+
+>It meets these criteria:
+
+>* Explicitly mentions running the command (this isn’t always obvious)
+* **Uses “run”** (and not “type”, “execute”, etc -- we want consistency)
+* Starts with the key information that describes the command, e.g. “To do X...enter Y command:”, “Do X by entering Y command:”
+* As short as possible
+
+> If you must deviate from the standard line, ensure you still meet the above criteria.
+
+=== ":white_check_mark: Correct"
+    Create the service by running:
```

Comment:
I think we'll just forever disagree but I left comments on the original doc for this. I'd prefer we had "by entering the command" than "running". I think this is clearer if we ever get to the point of translating docs etc but I think there should be some user research here to see what's preferred for those with screenreaders and non-English speakers

Reply:
I think @RichardJJG added this, any feedback here? 

I don't have a strong opinion here

Reply:
In my opinion, the word "run" in the context of running commands would be known to the audience (ie people who are familiar with using command line interfaces). This is a commonly used phase. 

Maybe a compromise would be to say "by running the command" to make sure that the word "running" has context?

Reply:
I am +1 on this @snneji

Reply:
@omerbensaadon let's table this and discuss at the DUX mtg since it seems mostly like just an opinion based thing we need to decide between the group?

Reply:
Actually, you know what, let's just go with Samia's suggestion for the sake of not wasting more time on this. 
> Maybe a compromise would be to say "by running the command" to make sure that the word "running" has context?

I think this works OK